src/gravel/controllers/nodes/deployment.py

```diff
@@ -13,10 +13,34 @@
 
 
 from enum import Enum
+from logging import Logger
+from pathlib import Path
+from uuid import UUID
 from pydantic import BaseModel, Field
+from fastapi.logger import logger as fastapi_logger
 
 from gravel.controllers.errors import GravelError
 from gravel.controllers.gstate import GlobalState
+from gravel.controllers.nodes.conn import ConnMgr
+
+from gravel.controllers.nodes.errors import (
+    NodeAlreadyJoiningError,
+    NodeBootstrappingError,
+    NodeHasBeenDeployedError,
+    NodeHasJoinedError
+)
+from gravel.controllers.nodes.messages import (
```

Comment:
(nit) rename this `gravel.controllers.nodes.models` ?

Reply:
well, they are models, true, but they are used as messages. I think for clarity it's better to have that as 'messages' and, eventually, move common models to a 'models' file.